Dockerfile.e2e

```diff
@@ -15,26 +15,29 @@ WORKDIR /go/src/github.com/docker/docker/
 
 # Generate frozen images
 COPY contrib/download-frozen-image-v2.sh contrib/download-frozen-image-v2.sh
-RUN contrib/download-frozen-image-v2.sh /output/docker-frozen-images \
+RUN ./contrib/download-frozen-image-v2.sh /output/docker-frozen-images \
 	buildpack-deps:jessie@sha256:dd86dced7c9cd2a724e779730f0a53f93b7ef42228d4344b25ce9a42a1486251 \
 	busybox:1.27-glibc@sha256:8c8f261a462eead45ab8e610d3e8f7a1e4fd1cd9bed5bc0a0c386784ab105d8e \
 	debian:jessie@sha256:287a20c5f73087ab406e6b364833e3fb7b3ae63ca0eb3486555dc27ed32c6e60 \
 	hello-world:latest@sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c
 
-# Download Docker CLI binary
-COPY hack/dockerfile hack/dockerfile
-RUN hack/dockerfile/install.sh dockercli
+# Install dockercli
+# Please edit hack/dockerfile/install/<name>.installer to update them.
+COPY hack/dockerfile/install hack/dockerfile/install
+RUN for i in dockercli; \
```

Comment:
Why the loop?

Reply:
Because of `copy/paste` 😅

Reply:
Oh I see I missed a directory when adding that install.sh to this one.